packages/definitions-parser/src/lib/definition-parser.ts

```diff
@@ -399,6 +407,10 @@ Other d.ts files must either be referenced through index.d.ts, tests, or added t
   }
 }
 
+function isRelativePath(path: string) {
+  return path.split(/\//).every(part => part.length > 0 && !part.match(/^\.+$|[\\\n\r]/));
```

Comment:
I get what the point is, but this returns true for absolute paths, it looks like

Reply:
I extracted it verbatim from https://github.com/DefinitelyTyped/dt-mergebot/blob/b8d318e166bc44484242cdf0bad7fea8c020fa45/src/pr-info.ts#L418-L420 and I think it works for absolute paths, because `"/absolute".split(/\//)` -> `["", "absolute"]`, so `"/absolute".split(/\//).every(part => part.length > 0)` -> `false`. Unless I'm missing something?

Reply:
Ah, I forgot the first element would be an empty string. Works for me.

Reply:
Thanks!